### PR TITLE
[deprecated - do not merge] PostgreSQL database

### PR DESCRIPTION
**Pull request by Kmaschta**
## TODO
- [x] DB lib structure
- [ ] Pure fonctionnal struct
- [ ] CRUD queries
- [ ] Natural query response
- [ ] And tests

### CODE REVIEW

File: api/src/lib/db/client.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import pg from 'pg';
-import config from 'config';
 
 export default config => {
```
Comment (djhi) [inline api/src/lib/db/client.js]:
```js
export default ({host, user, pswd, name}) => {
```

**Comment by jpetitcolas** [comment]
Perhaps also add node-migrat for migrations?

**Comment by Kmaschta** [comment]
@jpetitcolas Sure !

File: api/src/lib/db/model.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import manager from './manager';
+import validators from './validators';
+
+/**
+ * Usage:
+ *  const myModel = new Model(dbClient 'mymodel');
```
Comment (Kmaschta) [inline api/src/lib/db/model.js]: Outdated

**Pull request closed without merging**